lib/utils.js

```diff
@@ -179,103 +122,8 @@ exports.filterChange = function filterChange(opts) {
   };
 };
 
-// Preprocess documents, parse their revisions, assign an id and a
-// revision for new writes that are missing them, etc
-exports.parseDoc = function (doc, newEdits) {
-
-  var nRevNum;
-  var newRevId;
-  var revInfo;
-  var error;
-  var opts = {status: 'available'};
-  if (doc._deleted) {
-    opts.deleted = true;
-  }
-
-  function parseRevisionInfo() {
-    revInfo = /^(\d+)-(.+)$/.exec(doc._rev);
```

Comment:
simplified the regex a bit. no need for the capture groups or `(.+)$` to search all the way to the end of the string